test/integration/buyers/service_contracts_controller_test.rb

```diff
@@ -74,10 +74,29 @@ def test_unsubscribe_developers_from_service_with_two_no_suspended_applications
     assert_match I18n.t('service_contracts.unsubscribe_failure'), flash[:error]
   end
 
-  def test_unauthorized_access_master_on_premises
+  test 'unauthorized access master_on_premises' do
     login! master_account
     ThreeScale.stubs(master_on_premises?: true)
     get admin_buyers_account_service_contracts_path(account_id: @provider.id)
     assert_response :forbidden
   end
+
+  test 'renders a prompt if there is no default service plan' do
+    login_provider @provider
```

Comment:
This shouldn't be necessary because in the setup there is this: https://github.com/3scale/porta/blob/e13572517c900d6f0abe320918d599a62f2d1afb/test/integration/buyers/service_contracts_controller_test.rb#L19
in fact we can change [these 2 lines](https://github.com/3scale/porta/blob/e13572517c900d6f0abe320918d599a62f2d1afb/test/integration/buyers/service_contracts_controller_test.rb#L18-L19) of the setup for just `login! provider`

Reply:
🥇